packages/ecc-utils-design/src/components/form/tests/form.test.ts

```diff
@@ -0,0 +1,497 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-unused-expressions */
+
+// REMEMBER TO RE-ENABLE LINTING FOR TESTS
+
+import { expect } from "@open-wc/testing";
+import _ from "lodash-es";
```

Comment:
either way works fine.

Reply:
yeah it works fine, the issue is since there is no default export from the library so might consider it as error (on of my IDE extensions trips and throws error). This is not just for this but for some of the files in design as well.

Reply:
@anuragxxd maybe you can weigh in here!